io/pci/PowerVMEEH.py

```diff
@@ -73,6 +75,8 @@ def test_eeh_basic_pe(self):
         Test to execute basic error injection on PE
         """
         for self.addr in pci.get_pci_addresses():
+            self.num_of_miss = 0
+            self.num_of_hit = 0
```

Comment:
yes, because for each and every adapter, hit and miss to be set as zero before starting the test.